### PR TITLE
Add rawkv checksum doc

### PR DESCRIPTION
**Pull request by haojinming**
### What is changed?

Add rawkv checksum doc.

### Which version does your change affect?

dev

preview is [here](https://deploy-preview-329--tikv.netlify.app/)

### CODE REVIEW

**Comment by haojinming** [comment]
@pingyu @zeminzhou PTAL~ thanks

File: content/docs/dev/develop/rawkv/checksum.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        parent: RawKV-dev
+        weight: 2
+        identifier: Checksum-dev
+---
+
+This document walks you through how to use RawKV's `Checksum` API.
+
+`Checksum` API returns `Crc64Xor`, `TotalKvs` and `TotalBytes` from TiKV cluster.
+- `Crc64Xor`: The [XOR](https://en.wikipedia.org/wiki/Exclusive_or) of every key-value pair's [crc64](https://en.wikipedia.org/wiki/Cyclic_redundancy_check) value.
+- `TotalKVs`: The count of key-value pairs.
```
Comment (pingyu) [inline content/docs/dev/develop/rawkv/checksum.md:15]:
```suggestion
- `TotalKVs`: The number of key-value pairs.
```

File: content/docs/dev/develop/rawkv/checksum.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+- `TotalBytes`: The size of key-value pairs in bytes.
+
+_Note: If [API V2](../../../concepts/explore-tikv-features/api-v2) is enabled in tikv cluster, tikv client will encode prefix for raw key [here](https://github.com/tikv/client-go/blob/9c0835c80eba6cbda6fc4ae460d645de9d36cd05/internal/client/api_version.go#L57), and `Checksum` API also counts prefix into `Crc64Xor` and `TotalBytes`._
```
Comment (pingyu) [inline content/docs/dev/develop/rawkv/checksum.md:18]:
```suggestion
*Note: If [API V2](../../../concepts/explore-tikv-features/api-v2) is enabled, a `4` bytes prefix is encoded with keys, and also calculated by `Checksum` API*
```

File: content/docs/dev/develop/rawkv/checksum.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+## Go
+
+### Checksum with range
+
+Using the `Checksum` API, you can get `{Crc64Xor, TotalKvs, TotalBytes}` from TiKV in a range (from a `startKey` to an `endKey`).
```
Comment (pingyu) [inline content/docs/dev/develop/rawkv/checksum.md:24]:
```suggestion
Using the `Checksum` API, you can get `{Crc64Xor, TotalKvs, TotalBytes}` of a range from `startKey` (inclusive) to `endKey` (exclusive).
```

File: content/docs/dev/develop/rawkv/checksum.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+{{< info >}}
+`startKey` is inclusive while `endKey` is exclusive.
```
Comment (pingyu) [inline content/docs/dev/develop/rawkv/checksum.md:27]:
```suggestion
To calculate checksum of all keys, specify `startKey` and `endKey` as `[]byte("")`.
```

File: content/docs/dev/develop/rawkv/checksum.md [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+
+	checksum, err := cli.Checksum(ctx, []byte("k1"), []byte("k6"))
+	if err != nil {
+		panic(err)
+	}
+
+	fmt.Printf("Get checksum=, Crc64Xor:%d, TotalKvs:%d, TotalBytes:%d.\n",
+		checksum.Crc64Xor, checksum.TotalKvs, checksum.TotalBytes)
+}
+```
```
Comment (pingyu) [inline content/docs/dev/develop/rawkv/checksum.md:68]: Suggest to also write down the running result of the demo code. To give readers a direct impression of "checksum".

File: content/docs/dev/develop/rawkv/checksum.md [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+If you want to check all the data, the `startKey` and `endKey` can be specified as `[]byte("")`.
```
Comment (pingyu) [inline content/docs/dev/develop/rawkv/checksum.md:69]:
```suggestion
```

File: content/docs/dev/develop/rawkv/checksum.md [hunk 7]
```diff
@@ -0,0 +1,69 @@
+---
+title: Checksum
+description: Learn how to use RawKV's Checksum API.
+menu:
+    "dev":
+        parent: RawKV-dev
+        weight: 2
```
Comment (pingyu) [inline content/docs/dev/develop/rawkv/checksum.md:7]:
```suggestion
        weight: 5
```

To keep the same order as in introduction. Please do a favor to fix the order of "TTL" and "CAS".

**Comment by haojinming** [comment]
@pingyu all comments are addressed,PTAL~ thanks.

**Review by pingyu — APPROVED** [review]
LGTM~

**Review by zeminzhou — APPROVED** [review]
LGTM

**Pull request merged**